src/lake/tests/init/test.sh

```diff
@@ -34,6 +40,14 @@ $LAKE1 build
 .lake/build/bin/hello
 popd
 
+# Test math template
+
+$LAKE new qed math
+# Remove the require, since we do not wish to download mathlib during tests
+sed_i '/^require.*/{N;d}' qed/lakefile.lean
```

Comment:
Won't this just delete the first line? The `require mathlib` line is broken in the middle in the template.

Reply:
It should delete both the `N;` reads an additional line. See [this StackOverflow question](https://stackoverflow.com/questions/28021351/sed-remove-multiline-patterns-regexp) for an explanation. Also, it worked locally.